apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/layout.tsx

```diff
@@ -53,6 +52,15 @@ export default async function ChainPageLayout({
   params: { chain_id: string };
 }) {
   const chain = await getChain(params.chain_id);
+
+  // Enable faucet for all testnet chains.
+  if (chain.testnet) {
+    chain.services.push({
+      service: "faucet",
+      enabled: true,
+    });
+  }
```

Comment:
please do not mutate the array, instead you can do `chain.services = [...chain.services, <faucet>]`

Reply:
done